### ISSUE TITLE
Dig command is not working until reboot.

### ISSUE THREAD

**Issue by Markmaster** [blinksh/blink#1990]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

Blink version: 17.2.1.866
IOS: iOS 17.4 (21E219)
iPhone 15 Pro

### Describe the bug

When I try to use a dig command locally the app just replay that can't reach the server. The same is with the nslookup. The dig/nslookup on the remote servers are working as expected.

Dig command suppose to show my DNS setup.

![image](https://github.com/blinksh/blink/assets/4227967/ea0184a6-12fb-4cef-bf26-2c3f4cde5924)
![image](https://github.com/blinksh/blink/assets/4227967/aea4df3a-f567-44a2-ab4c-017d938991b4)

**Comment by carloscabanero** [comment]
Thanks for the report @Markmaster! I couldn't replicate it, but at the same time, we just updated the ios_system tools for the next Community / TestFlight, which I guess will be towards 17.2.3. So let's keep an eye on it and let me know if you see anything there.

I don't remember, were you added to Community already?

**Comment by carloscabanero** [comment]
This should have been fixed on 17.3.0. Please let us know.

**Comment by Markmaster** [comment]
@carloscabanero 17.2.2.868 is still the latest version for me from the AppStore. 
I also noticed that for me, rebooting no longer solves the issue. I also noticed that this problem is mostly present when using mobile internet. On wifi more often than not, this problem is not present. I thought that my ISP was blocking UDP (actually my ISP has some UDP issues for sure), but then I checked a few commands and they work with and without the tcp parameter. I've been using this mobile provider and the app for the last two years, but I'm only now encountering problems with the DIG command.![image](https://github.com/blinksh/blink/assets/4227967/e61fb774-a5aa-4e04-9270-cc935572ec40)
